src/crimson/os/seastore/cache.h

```diff
@@ -517,6 +517,14 @@ class Cache {
     return epm.get_block_size();
   }
 
+  bool get_checksum_needed(paddr_t paddr) const {
+    if (paddr.is_record_relative()) {
+      // we don't support checksum offload for journaling
+      return true;
+    }
```

Comment:
I added this function to determine whether checksum offload is supported for journal write. Do we need to make this an interface in `Journal`?

Reply:
(doesn't look like Cache needs to be involved/related here)

> Do we need to make this an interface in Journal?

Seems to me reasonable, because "record-relative" means the extent will be written inline (see how the decision is made by EPM), and `Journal` knows which device it belong to.

These fresh extents will be written directly to the device in journal, so crc offloading looks helpful.

Reply:
I have updated the commits to address your comment. Can you take a look?